src/pydata_sphinx_theme/assets/styles/sections/_header.scss

```diff
@@ -100,3 +109,25 @@
   padding-left: 2px;
   vertical-align: middle;
 }
+
+label.primary-toggle {
+  margin: 0 1em;
+  position: absolute;
+  left: 0;
+  top: calc(var(--pst-header-height) / 4);
+
+  @include media-breakpoint-up(md) {
+    display: none;
+  }
+}
+
+label.secondary-toggle {
```

Comment:
For both of these labels, we should set the `hover` action to be a "finger" for the mouse cursor

Reply:
[b74cd04](https://github.com/pydata/pydata-sphinx-theme/pull/726/commits/b74cd049d54fded0d067e6f0409295a7115b08f1)